app/models/ability.rb

```diff
@@ -59,6 +59,12 @@ def admin_perms
     can [:update, :destroy], TeamUser, team_id: @context_team.id
     can :duplicate, Team, :id => @context_team.id
     can :set_privacy, Project, :team_id => @context_team.id
+    can :cud, Feed do |obj|
+      team_ids = obj.feed_teams.map(&:team_id)
+      team_ids << obj.team_id
```

Comment:
Suggestion: Implement a method `team_ids` in `Feed` that contains this logic... I saw this being used elsewhere.

Reply:
Done